modules/connected-to-disconnected-config-registry.adoc

```diff
@@ -0,0 +1,184 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/connected-to-disconnected.adoc
+
+[id="connected-to-disconnected-config-registry_{context}"]
+= Configuring the cluster for the mirror registry
+
+After creating the mirror registry and mirroring the images to the registry, you need to make some modifications to your cluster in order for pods to be able to pull images from the mirror registry. 
```

Comment:
This sentence is a bit long. Would this change things too much?

"After creating and mirroring the images to the mirror registry, you must modify your cluster so that pods can pull images from the mirror registry." 

This is right at the ISG's sentence length limit of 25 words.